openfeature-provider/src/main/java/com/spotify/confidence/ConfidenceFeatureProvider.java

```diff
@@ -28,11 +28,23 @@ public class ConfidenceFeatureProvider implements FeatureProvider {
 
   private final Confidence confidence;
 
+  /**
+   * ConfidenceFeatureProvider constructor
+   *
+   * @param confidenceBuilder a Confidence.Builder
+   */
+  public ConfidenceFeatureProvider(Confidence.Builder confidenceBuilder) {
+    this.confidence = confidenceBuilder.buildForProvider();
+  }
```

Comment:
Nice! Does it mean we can make `buildForProvider` private and remove the related instructions from the README?